lib/dul_hydra/batch/models/batch_object_attribute.rb

```diff
@@ -0,0 +1,18 @@
+module DulHydra::Batch::Models
+
+  class BatchObjectAttribute < ActiveRecord::Base
+    belongs_to :batch_object, :inverse_of => :batch_object_relationships
+
+    # replace "adminMetadata" with constant Ddr::Datastreams::ADMIN_METADATA when available
+    ATTRIBUTE_DATASTREAMS = [ "adminMetadata", Ddr::Datastreams::DESC_METADATA, Ddr::Datastreams::STRUCT_METADATA ]
```

Comment:
Just to check why this is here ... means updating the code to accommodate new ds.